PivotCalibration/Resources/UI/qSlicerPivotCalibrationModule.ui

```diff
@@ -152,18 +150,43 @@
       <item>
        <widget class="QPushButton" name="startSpinButton">
         <property name="enabled">
-         <bool>false</bool>
+         <bool>true</bool>
         </property>
         <property name="text">
          <string>Start Spin Calibration</string>
         </property>
        </widget>
       </item>
       <item>
-       <widget class="QCheckBox" name="snapCheckBox">
-        <property name="text">
-         <string>Snap rotation to right-angle</string>
+       <widget class="ctkCollapsibleGroupBox" name="settingsGroupBox">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
         </property>
+        <property name="title">
+         <string>Settings</string>
+        </property>
+        <property name="checked">
+         <bool>false</bool>
+        </property>
+        <layout class="QVBoxLayout" name="verticalLayout_3">
+         <item>
+          <widget class="QCheckBox" name="snapCheckBox">
+           <property name="text">
+            <string>Snap rotation to right-angle</string>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QPushButton" name="flipButton">
```

Comment:
Please add some explanation to the button tooltip about when this button should be used.